src/components/remotecontrol/remotecontrol.css

```diff
@@ -111,42 +178,176 @@
         -webkit-box-align: center;
         -webkit-align-items: center;
         align-items: center;
+        width: 100%;
+        height: calc(100% - 4.2em);
     }
 
     .nowPlayingPageTitle {
-        text-align: center;
-        margin: 0.5em 0 0.75em;
+        /* text-align: center; */
+        margin: 0;
+    }
+
+    .nowPlayingInfoContainerMedia {
```

Comment:
> 
> 
> Why align the text to center? See what I have in mobile with long texts:
> ![image](https://user-images.githubusercontent.com/10274099/79011998-93cfbd80-7b65-11ea-94cd-a27b390e3a21.png)
> 
> With the text centered and the container justified and aligned to the center:
> 
> ![image](https://user-images.githubusercontent.com/10274099/79011989-8f0b0980-7b65-11ea-8eb6-a1cb4b238592.png)

This is a error. I didn't test with very long text. You see it center because of the class .emby-button. 

I should add 

> .nowPlayingAlbum a,
> .nowPlayingArtist a, {
>     text-align: left !important;
> }

Reply:
@samuel9554 That's much better. Last class doesn't have a comma. So it would be like this:

```
.nowPlayingAlbum a,
.nowPlayingArtist a {
text-align: left !important;
}
```

Reply:
Solved by ``dbdfd25``